docs/development/docguide.rst

```diff
@@ -11,6 +11,22 @@ keep to these as closely as possible, and we encourage affiliated packages to
 also adhere to these as they encourage useful documentation, a characteristic
 often lacking in professional astronomy software.
 
+Adding a Git commit
+===================
+
+When your changes only affect documentation (i.e., docstring or RST files)
+and do not include any code snippets that require doctest to run, you may
+add a ``[skip travis]`` in your commit message. For example::
+
+    git commit -m "Update documentation about this and that [skip travis]"
+
+When this commit is pushed out to your branch associated with a pull request,
+Travis CI will be skipped because it is not required. This is because the
+CI job to build the documentation resides in CircleCI.
+
+This special ``[skip travis]`` directive is being handled by ``ci-helpers``
+that Astropy uses for its continuous integration.
```

Comment:
This is now natively provided by travis, we in fact need to remove it from ci-helpers :)

```suggestion
```

Reply:
Good to know, thanks!